packages/blockchain-wallet-v4-frontend/src/components/Badge/BadgePlacement/BadgePlacement.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react'
+import { Icon } from '@blockchain-com/constellation'
+import { IconActivity } from '@blockchain-com/icons'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import { IconBadge } from '../IconBadge'
+import { BadgePlacement, BadgePlacementComponent } from '.'
+
+export default {
+  argTypes: {
+    placement: {
+      defaultValue: 'end'
+    },
+    shape: {
+      defaultValue: 'square'
+    }
+  },
+  component: BadgePlacement,
+  title: 'Components/Badge/BadgePlacement'
+} as ComponentMeta<BadgePlacementComponent>
+
+const BoxContent: FC = () => (
+  <div style={{ background: 'grey', display: 'inline-block', height: 64, width: 64 }} />
```

Comment:
too many inline styling, might be better to abstract them inside a styled component